ansible/roles/nginx/vars/cas_ssl.yml

```diff
@@ -4,19 +4,22 @@ nginx_ssl_key:  "{{ cas_nginx_ssl_key }}"
 
 nginx_sites:
 - server:
+   balancers:
+    - name: "{{ deploy_env }}_cas_commcare"
+      hosts: webworkers
+      port: "{{ django_port }}"
    file_name: "{{ deploy_env }}_cas_commcare"
    listen: "443 ssl default_server"
    server_name: "{{ CAS_SITE_HOST }}"
    client_max_body_size: 100m
-   balancer: webworkers
    proxy_set_headers:
    - "Host $http_host"
    - "X-Forwarded-For $remote_addr"
    add_header: "X-Frame-Options SAMEORIGIN"
    access_log: "{{ log_home }}/{{ deploy_env }}-timing.log timing"
    location1:
     name: /
-    balancer: webworkers
+    balancer: "{{ deploy_env }}_cas_commcare"
```

Comment:
balancer now specifies the name of the balancer you'd like to use with that location